tests/Unit/Donations/LegacyListeners/DispatchGiveRecurringAddSubscriptionPaymentAndRecordPaymentTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace GiveTests\Unit\Donations\LegacyListeners;
+
+use Give\Subscriptions\Models\Subscription;
+use GiveTests\TestCase;
+use GiveTests\TestTraits\RefreshDatabase;
+use InvalidArgumentException;
+
+/**
+ * @unreleased
+ */
+class DispatchGiveRecurringAddSubscriptionPaymentAndRecordPaymentTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /**
+     * @unreleased
+     * @return void
+     */
+    public function testShouldNotTriggerActionHookForSubscriptionInitialDonation()
+    {
+
```

Comment:
Please remove this space and make sure the file is properly formatted. 👍

Reply:
resolved